utils/_context/_scenarios.py

```diff
@@ -1695,6 +1711,10 @@ class scenarios:
         "K8S_LIB_INJECTION_FULL", doc=" Kubernetes Instrumentation complete scenario"
     )
 
+    installer_auto_injection = InstallerAutoInjectionScenario(
+        "INSTALLER_AUTO_INJECTION", doc="Installer auto injection scenario (minimal test scenario)"
```

Comment:
I would've loved to call it `SIMPLE_INSTALLER_AUTO_INJECTION` but we're hitting the max AMI name size (128 chars) 😅